app/src/examples/RuntimeTests/RuntimeTestsExample.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import RuntimeTestsRunner from './ReanimatedRuntimeTestsRunner/RuntimeTestsRunner';
+
+// load tests
+import './tests/Animations.test';
+// import './tests/sharedValue.test';
+// import './tests/withTiming.test';
```

Comment:
I removed the comments and disable some tests temporarily since they require rewriting. However, including navigation here might not be the best approach. We should find a better solutions. Let's address this in another pull request.